Latest release breaks example on github pages

Following same process as previously to be able to test kinesis webrtc stream on example page deployed at https://awslabs.github.io/amazon-kinesis-video-streams-webrtc-sdk-js/examples/index.html does not work.

The credentials used already has IAM policy AmazonKinesisVideoStreamsFullAccess attached.

The stream fails with following line on log,
[2023-08-21T10:57:40.093Z] [ERROR] [VIEWER] Encountered error starting: {
  "message": "The account id xxxxxxxxxxxxxxxxxx is not allowed to access the resources",
  "code": "AccessDeniedException",
  "time": "2023-08-21T10:57:40.092Z",
  "requestId": "xxxxxxxxxxxxxxxxxx",
  "statusCode": 403,
  "retryable": false,
  "retryDelay": 87.08014669943952
}

Able to produce locally when running through ```npm run develop```
Previous release seems to be working with same setup locally.

--- a/examples/master.js
+++ b/examples/master.js
@@ -74,23 +74,30 @@ async function startMaster(localView, remoteView, formValues, onStatsReport, onR
 
         const protocols = ['WSS', 'HTTPS'];
 
-        const describeMediaStorageConfigurationResponse = await kinesisVideoClient
-            .describeMediaStorageConfiguration({
-                ChannelARN: master.channelARN,
-            })
-            .promise();
-        const mediaStorageConfiguration = describeMediaStorageConfigurationResponse.MediaStorageConfiguration;
+        if (formValues.region?.toLowerCase() === 'us-west-2') {
+            console.log('[MASTER] Attempting to use media ingestion feature.');
+            const describeMediaStorageConfigurationResponse = await kinesisVideoClient
+                .describeMediaStorageConfiguration({
+                    ChannelARN: master.channelARN,
+                })
+                .promise();
+            const mediaStorageConfiguration = describeMediaStorageConfigurationResponse.MediaStorageConfiguration;
 
-        const mediaServiceMode = mediaStorageConfiguration.Status === 'ENABLED' || mediaStorageConfiguration.StreamARN !== null;
-        if (mediaServiceMode) {
-            if (!formValues.sendAudio || !formValues.sendVideo) {
-                console.error('[MASTER] Both Send Video and Send Audio checkboxes need to be checked to ingest and store media.');
-                return;
+            const mediaServiceMode = mediaStorageConfiguration.Status === 'ENABLED' || mediaStorageConfiguration.StreamARN !== null;
+            if (mediaServiceMode) {
+                if (!formValues.sendAudio || !formValues.sendVideo) {
+                    console.error('[MASTER] Both Send Video and Send Audio checkboxes need to be checked to ingest and store media.');
+                    return;
+                }
+                protocols.push('WEBRTC');
+                master.streamARN = mediaStorageConfiguration.StreamARN;
+                console.log(`[MASTER] Using media ingestion feature. Stream ARN: ${master.streamARN}`);
+            } else {
+                console.log('[MASTER] Not using media ingestion feature.');
+                master.streamARN = null;
             }
-            protocols.push('WEBRTC');
-            master.streamARN = mediaStorageConfiguration.StreamARN;
-            console.log(`[MASTER] Stream ARN: ${master.streamARN}`);
         } else {
+            console.log('[MASTER] Not using media ingestion feature.');
             master.streamARN = null;
         }
 

--- a/examples/viewer.js
+++ b/examples/viewer.js
@@ -126,15 +126,22 @@ async function startViewer(localView, remoteView, formValues, onStatsReport, onR
         const channelARN = describeSignalingChannelResponse.ChannelInfo.ChannelARN;
         console.log('[VIEWER] Channel ARN:', channelARN);
 
-        const mediaStorageConfiguration = await kinesisVideoClient
-            .describeMediaStorageConfiguration({
-                ChannelName: formValues.channelName,
-            })
-            .promise();
+        if (formValues.region?.toLowerCase() === 'us-west-2') {
+            console.log('[VIEWER] Using media ingestion feature');
+            const mediaStorageConfiguration = await kinesisVideoClient
+                .describeMediaStorageConfiguration({
+                    ChannelName: formValues.channelName,
+                })
+                .promise();
 
-        if (mediaStorageConfiguration.MediaStorageConfiguration.Status !== 'DISABLED') {
-            console.error('[VIEWER] Media storage and ingestion is ENABLED for this channel. Only the WebRTC Ingestion and Storage peer can join as a viewer.');
-            return;
+            if (mediaStorageConfiguration.MediaStorageConfiguration.Status !== 'DISABLED') {
+                console.error(
+                    '[VIEWER] Media storage and ingestion is ENABLED for this channel. Only the WebRTC Ingestion and Storage peer can join as a viewer.',
+                );
+                return;
+            }
+        } else {
+            console.log('[VIEWER] Not using media ingestion feature');
         }
 
         // Get signaling channel endpoints

